packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -760,10 +779,15 @@ class Splitter extends (CollectionWidget as any) {
       case 'onResize':
       case 'onItemCollapsed':
       case 'onItemExpanded':
+        this._updateNestedSplitterOption(name, value);
         this._createEventAction(name);
         break;
       case 'separatorSize':
         this._updateResizeHandlesOption(name, value);
+        this._updateNestedSplitterOption(name, value);
+        break;
+      case 'rtlEnabled':
```

Comment:
For rtlEnabled property we shouldn't do anything. Default logic will add dx-rtl class. We use rtlEnabled in calculation but we read this option onResize event, and there this value will have right value.

Reply:
In the parent components it call invalidate() and will call full rerender. It suits for all our components